backend/src/main/java/ca/bc/gov/hlth/mohums/user/UserAttributeEntity.java

```diff
@@ -0,0 +1,72 @@
+package ca.bc.gov.hlth.mohums.user;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "USER_ATTRIBUTE", schema = "KEYCLOAK")
+public class UserAttributeEntity {
```

Comment:
Correct. I did not check if it's available as a library, but I'm making small changes to those Entity classes (For example bidirectional relationship between UserEntity and UserRoleMapping - so that URM can be referenced from UserEntity when preparing JOIN statement.